modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -14,6 +14,17 @@
         @click="confirmAndCancel"
       />
     </template>
+    <template #localeDisplay>
```

Comment:
Must be added to https://github.com/apostrophecms/doc-template-library/blob/fd44c2093c9277bbfc46e7ec945db4e142ab90a8/ui/apos/components/AposTemplateLibraryEditor.vue#L127

Reply:
Adding a note to myself, but not sure we need a locale switcher for doc template libraries.